Configurations/00-base-templates.conf

```diff
@@ -28,7 +28,7 @@ my %targets=(
 	build_file	=> "Makefile",
 
 	AR		=> "ar",
-	ARFLAGS		=> "r",
+	ARFLAGS		=> "qc",
```

Comment:
Oh wait, this is the wrong spot to make this change, it should really have been made in the `BASE_unix` section further down...  where it's already "qc", so no change needed.

You might optionally want to set AR and ARFLAGS here to the empty string,

Reply:
Ah, so to avoid such confusion in the future I set the values of  AR and ARFLAGS in the general section to "(unused)".
Would you like to (re-)approve this, @levitte?